buildconfig/BuildConfig.cpp.in

```diff
@@ -62,8 +62,10 @@ Config::Config()
     UPDATER_BASE = "@Launcher_UPDATER_BASE@";
 
     GIT_COMMIT = "@Launcher_GIT_COMMIT@";
+    GIT_TAG = "@Launcher_GIT_TAG@";
     GIT_REFSPEC = "@Launcher_GIT_REFSPEC@";
-    if (GIT_REFSPEC == QStringLiteral("GITDIR-NOTFOUND"))
+    if (GIT_REFSPEC == QStringLiteral("GITDIR-NOTFOUND")
+        || GIT_TAG == QStringLiteral("-128-NOTFOUND"))
     {
         VERSION_CHANNEL = QStringLiteral("stable");
```

Comment:
It's not really accurate for a build from a pull request to be labeled as part of the "stable" channel, though, is it? Although, is there a way to easily tell that a build was made from a PR as opposed to a release?

<img width="895" alt="Screen Shot 2022-06-22 at 6 55 06 PM" src="https://user-images.githubusercontent.com/79120643/175168092-3d70e33f-9827-421d-8e88-d2523dee46e3.png">

Either way, the channel looks like it's pretty much unused in PolyMC anyway, so 🤷🏼‍♂️

Reply:
oh hm.. I thought it only is -128-NOTFOUND, when there is no Git repo

Reply:
but now the tag's not hidden :/
<img width="230" alt="Screen Shot 2022-06-23 at 11 58 49 PM" src="https://user-images.githubusercontent.com/79120643/175459488-2c91e753-6d2c-41f9-92f9-c16150f85f74.png">

Reply:
Works fine for me. Did you tag the current commit locally?

Reply:
> Did you tag the current commit locally?

Yeah oops that was the problem, ignore me lol